common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -46,37 +49,55 @@ import whisk.core.entity.ArgNormalizer.trim
 sealed abstract class Exec(val kind: String) {
     def image: String
 
+    /**
+     * Method to calculate the size of the action.
+     * The size of the action is defined as the sum of sizes of everything, that is stored in the Exec(code, init, jar, main, ...).
+     *
+     * @return true if the exec has less characters than the sizelimit.
+     */
+    def size: ObjectSize
+
     override def toString = Exec.serdes.write(this).compactPrint
 }
 
 protected[core] case class NodeJSExec(code: String, init: Option[String]) extends Exec(Exec.NODEJS) {
     val image = "whisk/nodejsaction"
+    def size = (code sizeInBytes) + (init.getOrElse("") sizeInBytes)
```

Comment:
I decided to name the function `sizeInBytes` because the size of the string has to be calculated first. If the function is called bytes or getBytes, it sounds like the size is already calculated and stored with the String as an additional parameter.